pom.xml

```diff
@@ -165,6 +165,12 @@
             <version>${target.maven.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugin-tools</groupId>
+            <artifactId>maven-plugin-annotations</artifactId>
+            <version>3.6.4</version>
```

Comment:
${target.maven.version}

Reply:
> `${target.maven.version}`

IIUC project _Plugin Tools_ doesn't share lifecycle with _Maven_. 3.6.4 is currently the latest (https://repo.maven.apache.org/maven2/org/apache/maven/plugin-tools/maven-plugin-annotations/), while _Maven_ is 3.8.6.
There is no `maven-plugin-annotations:3.6.3` so using this property would not work.

Reply:
Aha! :)